platform/security/handler/security-handler-api/src/main/java/org/codice/ddf/security/handler/api/AnonymousAuthenticationToken.java

```diff
@@ -28,9 +31,20 @@
             BSTAuthenticationToken.BST_NS + BSTAuthenticationToken.TOKEN_VALUE_SEPARATOR
                     + BST_ANONYMOUS_LN;
 
-    public AnonymousAuthenticationToken(String realm) {
+    private String ipAddress;
```

Comment:
This change isn't entirely correct. We need to save the ip address down into the actual credentials so they'll be serialized during the SOAP call.